### DISCUSSION TITLE
Problem with AutoComplete

### DISCUSSION THREAD

**Discussion by meliora-solution** [microsoft/fluentui-blazor#2912] [General]
Problem with AutoComplete 

I have save a record of a customer for example :
Name : Jason B
State California
City San Francisco

Then When I loaded existing customer with the region

  SelectedState = new List<RegionDto> { Provinces.FirstOrDefault(p => p.Id == Input.StateId) };
  SelectedCity = new List<RegionDto> { Citiess.FirstOrDefault(k => k.Id == Input.CityId) };

The clear item is locked. I can not changed the data.

I found the problem, I think it is the new version has a problem because when I used older version, it works: 

```
<PackageReference Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.*-*" />
```
I used older version :
```
<PackageReference Include="Microsoft.FluentUI.AspNetCore.Components" Version="4.7.2" />
```
And the older version works fine. So I guess the new version has a problem ???

**Comment by dvoituron** [comment]
We already fixed it, for the next Release: #2903

**Reply by meliora-solution** [reply]
Thank you for letting me know.

**Comment by meliora-solution** [comment]
I changed to new component and it works well now. Thank you.
